test/legacy_test/test_instance_norm_op_v2.py

```diff
@@ -12,15 +12,69 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
 
 import numpy as np
 from eager_op_test import OpTest, convert_float_to_uint16
 
 import paddle
-import paddle.nn.functional as F
-from paddle import fluid, nn
-from paddle.fluid import Program, core, framework, program_guard
+from paddle import fluid
+from paddle.fluid import Program, core, program_guard
+
+
+def instance_norm_wrapper(
+    input, weight, bias, epsilon=1e-5, momentum=0.9, data_format='NCHW'
+):
+    if data_format == "AnyLayout":
+        data_format = "NCDHW"
+    return paddle.nn.functional.instance_norm(
```

Comment:
why replace paddle._C_ops.instance_norm with F.instance_norm?

Reply:
it can't run to_static process.